src/Keypad.cpp

```diff
@@ -75,7 +75,7 @@ bool Keypad::getKeys() {
 		keyActivity = updateList();
 		startTime = millis();
 	}
-
+	
```

Comment:
I think that adding a tab to a blank line will fail the formatting tests. Also, doing a format will change the expression layout on line 73.